.github/workflows/ci.yml

```diff
@@ -24,6 +24,13 @@ jobs:
             database_image: "mariadb:10"
             coverage: false
             experimental: true
+          - mediawiki_version: '1.40'
+            smw_version: dev-master
+            php_version: 8.1
+            database_type: mysql
+            database_image: "mariadb:11.2"
+            coverage: true
+            experimental: true
```

Comment:
for all cases where tests are passing, experimental can be set to false

Reply:
@gesinn-it-gea done.